src/services/observation.service.ts

```diff
@@ -565,3 +577,30 @@ export const axGetPlantnetSuggestions = async (imageUrls, organs) => {
     return { success: false, data: [] };
   }
 };
+
+export const axGetGroupByDayCreatedOn = async (userId) => {
```

Comment:
both axGetGroupByDayCreatedOn and axGetGroupByDayCreatedOn functions same?

Reply:
I couldn't find a duplicate axGetGroupByDayCreatedOn function.